platform-sdk/platform-apps/tests/PlatformTestingTool/build.gradle.kts

```diff
@@ -15,31 +15,39 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.application")
-  id("com.swirlds.platform.protobuf")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.application")
+    id("com.google.protobuf")
 }
 
 dependencies {
-  // Individual Dependencies
-  compileOnly(libs.spotbugs.annotations)
-  implementation(project(":swirlds-merkle"))
-  implementation(libs.commons.math3)
-  implementation(libs.commons.io)
-  implementation(libs.protobuf)
+    // Individual Dependencies
+    compileOnly(libs.spotbugs.annotations)
+    implementation(project(":swirlds-merkle"))
+    implementation(libs.commons.math3)
+    implementation(libs.commons.io)
+    implementation(libs.protobuf)
 
-  // Bundle Dependencies
-  implementation(libs.bundles.logging.impl)
+    // Bundle Dependencies
+    implementation(libs.bundles.logging.impl)
 
-  // Test Dependencies
+    // Test Dependencies
 
-  // These should not be implementation() based deps, but this requires refactoring to eliminate.
-  implementation(project(":swirlds-unit-tests:common:swirlds-test-framework"))
-  implementation(project(":swirlds-unit-tests:common:swirlds-common-test"))
-  implementation(project(":swirlds-unit-tests:structures:swirlds-merkle-test"))
+    // These should not be implementation() based deps, but this requires refactoring to eliminate.
+    implementation(project(":swirlds-unit-tests:common:swirlds-test-framework"))
+    implementation(project(":swirlds-unit-tests:common:swirlds-common-test"))
+    implementation(project(":swirlds-unit-tests:structures:swirlds-merkle-test"))
 
-  testImplementation(testLibs.bundles.junit)
-  testImplementation(testLibs.bundles.mocking)
+    testImplementation(testLibs.bundles.junit)
+    testImplementation(testLibs.bundles.mocking)
 }
 
-tasks.withType<Javadoc>() { enabled = false }
+protobuf { protoc { artifact = "com.google.protobuf:protoc:3.21.5" } }
```

Comment:
We should make sure this is correct. We replaced google's protobuf compiler in the main codebase, so I would think the testing tool should use the same plugin.

Reply:
Right now, this is preserving what is on `develop`. Just moved things around.

Maybe we can address this concern in a follow up once this is merged?

Reply:
@jsync-swirlds I doubt we will ever convert PTT to use PBJ. PTT is in the process of being deprecated and removed. Additionally, PTT uses a small handful of PTT specific protobuf definitions.